package.json

```diff
@@ -87,8 +91,8 @@
     "lint": "tslint packages/*/lib/**/*.ts packages/*/test/**/*.ts --exclude '**/*.d.ts'",
     "build": "tsc",
     "doc": "typedoc --name Comunica --exclude \"**/+(node_modules|test|mocks)/**\" --disableOutputCheck --mode modules --out documentation $(find packages ! -name 'packager' -type d -mindepth 1 -maxdepth 1)",
-    "preinstall": "node -e \"if (!fs.existsSync('packages/runner-cli/bin/run.js')) { var os = fs.createWriteStream('packages/runner-cli/bin/run.js'); os.write('#!/usr/bin/env node\\nconsole.error(\\'Temporary runner script during lerna installation. If you see this, please run yarn install again.\\');\\nprocess.exit(1);'); os.end(); }\"",
-    "postinstall": "lerna bootstrap && tsc"
+    "preinstall": "node -e \"if (!fs.existsSync('packages/runner-cli/bin/run.js')) { var os = fs.createWriteStream('packages/runner-cli/bin/run.js'); os.write('#!/usr/bin/env node\\nconsole.error(\\'Temporary runner script during lerna installation. If you see this, please run yarn install again.\\');\\nprocess.exit(1);'); os.end(); }\" && node -e \"if (!fs.existsSync('packages/runner/bin/compile-config.js')) { var os = fs.createWriteStream('packages/runner/bin/compile-config.js'); os.write('#!/usr/bin/env node\\nconsole.error(\\'Temporary compile-config script during lerna installation. If you see this, please run yarn install again.\\');\\nprocess.exit(1);'); os.end(); }\"",
+    "postinstall": "lerna bootstrap --ignore-scripts && tsc && lerna bootstrap"
```

Comment:
No, this fixes the permissions issue that we were having. This makes sure that all packages are linked, after which we build, and then we do all of the other pre/postinstall stuff.

The alternative would be to run a single `lerna bootstrap` and move the `tsc` calls in each package separately, but this would actually be much slower (we did this in the past).